packages/calypso-build/webpack/util.js

```diff
@@ -67,4 +67,50 @@ function IncrementalProgressPlugin() {
 	return new webpack.ProgressPlugin( createProgressHandler() );
 }
 
-module.exports = { cssNameFromFilename, IncrementalProgressPlugin };
+const nodeModulesToTranspile = [
+	// general form is <package-name>/.
+	// The trailing slash makes sure we're not matching these as prefixes
+	// In some cases we do want prefix style matching (lodash. for lodash.assign)
+	'@github/webauthn-json/',
```

Comment:
Would now be a good time to extend this to support semver?

Reply:
The idea is to have semver specifiers like `debug@^3` in the whitelist and be able to read the `package.json` for a tested package and match the semver version?

That seems a big enough topic to warrant its own PR. We probably want to keep some cache of "file-path -> package-version" mappings, so that we don't do the look up for every file (every package can include a large number of contained files and webpack sees each sub-path separately).